datadog_checks_dev/datadog_checks/dev/tooling/configuration/consumers/example.py

```diff
@@ -161,7 +161,7 @@ def write_option(option, writer, indent='', start_list=False):
 
         if 'options' in option:
             multiple = option['multiple']
-            options = option['options']
+            options = sorted(option['options'], key=lambda opt: -opt['order'])
```

Comment:
Looks like python's stable sort will preserve the existing ordering of items that don't explicitly set an order attribute (using the default of `0`) - so this should be a noop for existing configs 👍

Reply:
Yes, exact :)